src/server/views/transition-page.ejs

```diff
@@ -1,139 +1,50 @@
-                z-index: 1000;
-            }
-
-            #card h3 {
-                font-size: 12px;
-                line-height: 1.5;
-                color: #384a51;
-            }
-
-            #card p {
-                font-size: 14px;
-                line-height: 1.43;
-                font-weight: 200;
-            }
-
-			@keyframes rotation {
-				from {
-					transform: rotate(0deg);
-				}
-
-				to {
-					transform: rotate(359deg);
-				}
-			}
-
-        </style>
-    </head>
-
-    <body>
-        <div id="top-container">
-            <div id="main-message">
-                <p>Check your browser to make sure you're visiting</p>
-                <h2><b>go.gov.sg/<%= shortUrl _%></b></h2>
-            </div>
-			<!-- loader circle -->
-			<div id="loader-circle">
-				<img src="https://gosg-public.s3-ap-southeast-1.amazonaws.com/icon-loader.svg"/>
-			</div>
-            <div id="card">
-                <h3>You will be redirected to your requested content in a few seconds:</h3>
-                <p><%= longUrl _%></p>
-            </div>
-
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="Content-Language" content="en">
+    <meta charset="UTF-8">
+    <title>Go.gov.sg</title>
+    <link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Sans:wght@400;500;700&display=swap" rel="stylesheet">
+    <link href="./assets/transition-page/styles/transition-page.css" rel="stylesheet">
+    <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon">
+</head>
+
+<body>
+    <main id="main-container">
+        <div id="main-message">
+            <a href="/">
+                <!-- go logo -->
+                <img src="./assets/transition-page/icons/go-logo.svg" draggable="false" />
+            </a>
+            <div class="divider"></div>
+            <h3>Check your browser to make sure you’re accessing a <span
+                    class="emphasize-text-color">gov.sg</span> link.
+            </h3>
+            <p>You are visiting a page at <span class="semi-bold-text"><%= rootDomain _%></span>.</p>
+            <button onclick="proceedToDestination()">
+                <div id="main-button-content">
+                    <p>Proceed to your page</p>
+                    <!-- button right arrow icon -->
+                    <img src="./assets/transition-page/icons/icon-arrow-right.svg" draggable="false" />
+                </div>
+            </button>
         </div>
-        <div id="bottom-container">
-            <img src="https://gosg-public.s3-ap-southeast-1.amazonaws.com/go-logo.svg"/>
+    </main>
+    <div id="page-banner">
+        <div id="page-banner-content">
+            <!-- banner info icon -->
+            <img src="./assets/transition-page/icons/icon-info.svg" draggable="false" />
+            <p>You will only be shown this page the first time you access this short link.</p>
```

Comment:
We may have to reword this sentence because not tracking the full history of the links visited. FYI  @yong-jie